specs/tendrl_host_listing_monitoring_summary_api.adoc

```diff
@@ -0,0 +1,187 @@
+// vim: tw=79
+
+= Host Listing Backend Support
+
+== Introduction
+
+The host listing page displays the following details per host:
+
+* Hostname and status
+* Storage used, total and percentage used
+* Cpu percentage used
+* Memory used, total and percentage used
+* Name of Cluster to which host belongs
+* Role of the node in the cluster
```

Comment:
@dahorak @shtripat I think @anivargi will have a spec to detail the other aspects of the host listing page. @anivargi can you please confirm the same